base/v0_3_exp/translate.go

```diff
@@ -387,7 +423,21 @@ func (c Config) addMountUnits(ret *types.Config) translate.TranslationSet {
 			continue
 		}
 		fromPath := path.New("yaml", "storage", "filesystems", i, "with_mount_unit")
-		newUnit := mountUnitFromFS(fs)
+		remote := false
+		// check filesystems targeting /dev/mapper devices against LUKS to determine if a
+		// remote mount is needed
+		if strings.HasPrefix(fs.Device, "/dev/mapper/") || strings.HasPrefix(fs.Device, "/dev/disk/by-id/dm-name-") {
```

Comment:
Hmm, wish there was a more elegant way to do this, but I don't have a better idea. Can we at least document this in `with_mount_unit`?